application/src/main/java/uk/nhs/digital/uec/api/config/SecurityConfig.java

```diff
@@ -17,6 +18,7 @@
 @Configuration
 @EnableWebSecurity
 @EnableGlobalMethodSecurity(prePostEnabled = true)
+@Profile("!local")
```

Comment:
This should be removed also ?

Reply:
SecurityConfig will not be executed for local profile as this will have the non local filter